var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -80,8 +80,12 @@ class Xsdk(Package):
     depends_on('alquimia@develop', when='@develop')
 
     depends_on('sundials@3.1.0~int64+hypre', when='@0.3.0')
-    depends_on('plasma', when='@0.3.0 %gcc@4.9:')
-    depends_on('magma', when='@0.3.0 +cuda')
+    depends_on('sundials@3.1.0~int64+hypre', when='@develop')
+
+    depends_on('plasma@17.1', when='@develop %gcc@4.9:')
+
+    depends_on('magma@2.2.0', when='@0.3.0 +cuda')
```

Comment:
you can write `@0.3.0:` here, which will be satisfied by `@develop` as well. So one line less.

Reply:
I'd like to keep '@develop' here so that its easy to figure out the '@0.3.0' dependencies vs '@develop' dependencies [for all these packages].

At some point I'm hoping to be able to switch to plasma@develop magma@develop etc for this xsdk@develop dependency